charts/portal/templates/_helpers.tpl

```diff
@@ -42,6 +42,8 @@
   {{- else }}
   value: {{ .Values.global.platform.membership.oauthClient.secret | quote }}
   {{- end }}
+- name: FEATURES_DISABLED
+  value: "console_v3_beta"
```

Comment:
Humm, you will not be able to enable the feature in staging.
You need to map the FEATURE_DISABLED env var to a yaml keyinside `values.yaml` and default it with "console_v3_beta" 

is feature disabled intended to be a list of elements ? like `value: "{{ join " " .Values.config.stack.minimalStackModules}}"`

Reply:
nop it is just a string